modules/ingest-common/src/test/java/org/opensearch/ingest/common/RemoveProcessorTests.java

```diff
@@ -57,7 +54,7 @@ public void testRemoveFields() throws Exception {
             false
         );
         processor.execute(ingestDocument);
-        assertThat(ingestDocument.hasField(field), equalTo(false));
+        assertFalse(ingestDocument.hasField(field));
```

Comment:
This change doesn't look like related to the issue, plus the reporting of the assertion failures by `assertThat` is much more readable than by `assertFalse`

Reply:
Sorry for missing explanation, that's because I see `assertThat` is deprecated, so I change it to `assertFalse` by the way, do you think we can still use `assertThat`?

Reply:
Yes, please use `org.hamcrest.MatcherAssert.assertThat`, thank you

Reply:
~Yes, please use `org.hamcrest.MatcherAssert.assertThat`, thank you~, sorry it is coming from OpenSearchTestCase, please keep it, we will update the base test case